src/commands/deploy.rs

```diff
@@ -712,3 +717,10 @@ fn print_available_routes(
         }
     }
 }
+
+fn check_trigger_is_supported(cfg: &RawAppManifest) -> Result<()> {
+    match cfg.info.trigger {
+        ApplicationTrigger::Redis(_) => bail!("Redis triggers are not supported"),
```

Comment:
Should we allow-list rather than deny-list?  For example if and when we implement a cron or queue trigger, allow-list might result in more correct behaviour if we forget to update the list...?

Reply:
Good call.

Reply:
Note that I just tried this and I get an "unreachable pattern" warning after exhausting all other possible options. So I think this is fine, because new trigger types will require an explicit addition to the pattern matcher.

Reply:
Ah!  For some reason I thought they were not all listed (and there was a default clause) but no, it's all explicit so 100% safe.  Sorry about that.